src/attribute.rs

```diff
@@ -1,19 +1,18 @@
-// Copyright 2017, The Gtk-rs Project Developers.
-// See the COPYRIGHT file at the top-level directory of this distribution.
-// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
-
-use glib::translate::*;
-use pango_sys;
-use AttrClass;
 use Attribute;
+use FontDescription;
 use Gravity;
 use GravityHint;
+use Language;
+use Rectangle;
 use Stretch;
 use Style;
 use Underline;
-use Variant;
 use Weight;
 
+use glib::translate::from_glib_full;
+use glib::translate::ToGlib;
+use glib::translate::ToGlibPtr;
+
 impl Attribute {
     #[cfg(any(feature = "v1_38", feature = "dox"))]
     pub fn new_background_alpha(alpha: u16) -> Option<Attribute> {
```

Comment:
This (and all the others too) should probably return an `AttributeBackgrondAlpha`?

Reply:
No, it's all casted as `Attribute` in the source code. Also, it's not really supposed to be "used" so better not give access to the fields.

Reply:
Ack